test/runtime/lb.go

```diff
@@ -301,6 +301,103 @@ var _ = Describe("RuntimeLB", func() {
 		status = vm.ContainerExec("client", testCmd)
 		status.ExpectSuccess("LB is not working after restart")
 	})
+
+	Context("Services Policies", func() {
+		BeforeEach(func() {
+			vm.PolicyDelAll()
+		})
+
+		AfterEach(func() {
+			vm.SampleContainersActions(helpers.Delete, helpers.CiliumDockerNetwork)
+			vm.PolicyDelAll()
+
+			status := vm.ExecCilium(fmt.Sprintf("config %s=false",
+				helpers.OptionConntrackLocal))
+			status.ExpectSuccess()
+		})
+
+		policiesTest := func() {
+
+			httpd1, err := vm.ContainerInspectNet(helpers.Httpd1)
+			Expect(err).Should(BeNil())
+
+			httpd2, err := vm.ContainerInspectNet(helpers.Httpd2)
+			Expect(err).Should(BeNil())
+
+			service1 := "2.2.2.100:80"
+			service2 := "2.2.2.101:80"
+			getHTTP := func(service, target string) string {
+				return helpers.CurlFail(fmt.Sprintf(
+					"http://%s/%s", service, target))
+			}
+
+			status := vm.ServiceAdd(100, service1, []string{
+				fmt.Sprintf("%s:80", httpd1["IPv4"])}, 1)
+			status.ExpectSuccess("L4 service cannot be created")
+
+			status = vm.ServiceAdd(101, service2, []string{
+				fmt.Sprintf("%s:80", httpd2["IPv4"])}, 1)
+			status.ExpectSuccess("L4 service cannot be created")
+
+			_, err = vm.PolicyImport(vm.GetFullPath(policiesL7JSON), helpers.HelperTimeout)
+			Expect(err).Should(BeNil())
+
+			By("Simple Ingress")
+			status = vm.ContainerExec(helpers.App1, getHTTP(service1, "public"))
```

Comment:
Please make "public" and "private" constants in `cons.go` and utilize them in this test.